### ISSUE TITLE
Is it possible to run on 4g vram gpu?

### ISSUE THREAD

**Issue by WillLam123** [oobabooga/text-generation-webui#1537]
I only have a laptop with gtx 2050 gpu but I really want to tried this out. Is it possible to run on 4g vram gpu? Or what else choice do I have.

**Comment by BarfingLemurs** [comment]
yes, you will need a 4bit quantized 7b model which should fit. 
make your settings like this: 
![image](https://user-images.githubusercontent.com/128182951/234308081-0582ac1d-7546-406c-b8b4-0de4873e5eb8.png)
tune your --pre-layer settings, i dont know what to set it to though

**Comment by WillLam123** [comment]
> 4bit quantized 7b model
Thank you so much! I will try it right away! 
Btw, what does this "quantized" and "4-bit" mean in the name? After googling it my guess is that 4-bit required less vram than 8-bit or 16-bit, is that correct?

**Comment by BarfingLemurs** [comment]
gtx 2050 = nvidia

>Btw, what does this "quantized" and "4-bit" mean in the name? After googling it my guess is that 4-bit required less vram than 8-bit or 16-bit, is that correct?

yes, get the .pt models that say 4bit from various websites or piracy

**Comment by WillLam123** [comment]
> gtx 2050 = nvidia
> 
> > Btw, what does this "quantized" and "4-bit" mean in the name? After googling it my guess is that 4-bit required less vram than 8-bit or 16-bit, is that correct?
> 
> yes, get the .pt models that say 4bit from various websites or piracy

Just realize that there are smaller models I can use. I am trying a 1.3b model to see if that can make it work.

**Comment by BarfingLemurs** [comment]
yes, startup with a small model, and then change your settings. I had to do this with the one click installers.
for example I had to select load-in-8bit, and then change to load a larger model. Its a hassle but it works. There might be settings to mess with later.

**Comment by CarlKenner** [comment]
Yes, it's possible. Some models will work out of the box without changing any settings, some won't work.
I have a GTX 970.

**Comment by WillLam123** [comment]
> 

Good to know that. Can you share some of the models that work without changing any settings?

**Comment by CarlKenner** [comment]
> Good to know that. Can you share some of the models that work without changing any settings?

facebook_galactica-1.3b
stabilityai_stablelm-tuned-alpha-3b (but it's slow)
RWKV-4-Pile-169M-20220807-8023.pth
RWKV-4-Pile-1B5-20220929-ctx4096.pth

But any of the models with 1.5 billion parameters or less will probably work.

Note that if you want to use Galactica or StableLM in instruct chat mode, you need to change the Character settings to match the way they were trained. I'm waiting on them to accept my pull request for that support to be built in.
